app/index.js

```diff
@@ -4,8 +4,9 @@ const { start } = require('./commands/start')
 const { all } = require('./commands/all')
 const { daily } = require('./commands/daily')
 const { stateName } = require('./commands/stateName')
+const { testingCentres } = require('./commands/testingCentres')
 
-require('dotenv').config()
+require('dotenv').config({ path: __dirname + '/../.env' })
```

Comment:
Is there any specific reason you made this change?

Reply:
I placed the .env file in the repo's base folder

Reply:
The old code was already picking it up from base folder. Correct me if I'm wrong here.

Reply:
It was, my bad. I'm reverting that change